pgcli/pgexecute.py

```diff
@@ -186,7 +185,9 @@ def connect(self, database=None, user=None, password=None, host=None,
 
             cursor = conn.cursor()
 
-        superuser = self._select_one(cursor, "select current_setting('is_superuser')::bool")[0]
+        cursor.execute("SHOW ALL")
```

Comment:
There might be a simpler way. For example, if we're on redshift, just assume is_superuser is False.